api/agent.go

```diff
@@ -16,6 +18,19 @@ type Agent struct {
 	region     string
 }
 
+// KeyringResponse is a unified key response and can be used for install,
+// remove, use, as well as listing key queries.
+type KeyringResponse struct {
```

Comment:
@schmichael The API package mimics the structs package largely but gives a separation between internal and external structs